README.md

```diff
@@ -14,7 +14,10 @@
 <br /><br />
 <a href="https://github.com/man-group/ArcticDB/actions"><img src="https://github.com/man-group/ArcticDB/actions/workflows/build.yml/badge.svg"/></a>
 </p>
+---
+## Star History
 
+[![Star History Chart](https://api.star-history.com/svg?repos=man-group/ArcticDB&type=Date)](https://star-history.com/#man-group/ArcticDB&Date)
```

Comment:
This is really big and takes over:

![image](https://github.com/man-group/ArcticDB/assets/3106180/9ce9a488-721d-4bf4-afd8-f8dd612e5971)

This IMO is way too big. Do we also want this to be on the front page? Bit vain in my eyes...